docs/kb/api-version-3/prospects.md

```diff
@@ -245,6 +248,65 @@ XML responses to `create` requests are identical to `update` and `read` requests
 
 <a name="14833-updating-field-values" id="updating-field-values"></a>
 
+## [](#enpoints-batch-processing-)Endpoints for Batch Processing
+
+There are 3 endpoints available for batch processing up to 50 prospects at a time:
+
+/api/prospect/version/3/do/batchCreate
+
+/api/prospect/version/3/do/batchUpdate
+
+/api/prospect/version/3/do/batchUpsert
+
+These endpoints expect a POST or GET variable called "prospects" which holds either JSON or XML encoded data.
```

Comment:
query param is probably the most appropriate word given the example
`These endpoints expect a query param called "prospects"...`

Later down though we should specify the request method. I would expect it to be a `POST` since we're doing an update but @codeian should verify this.